Dockerfile

```diff
@@ -27,10 +45,12 @@ RUN cat Gemfile.lock | grep -A1 "BUNDLED WITH" | tail -n1 | awk '{print $1}' > B
 RUN set -ex && \
   apk add --update --no-cache make gcc libc-dev mariadb-dev postgresql-dev sqlite-dev git && \
   apk upgrade && \
+  gem uninstall bundler -v 2.1.4 && \
```

Comment:
This didn't actually seem to help my local trivy report, but removing `/usr/local/lib/ruby/gems/${RUBY_VERSION}/specifications/default/bundler-2.1.4.gemspec` seemed to do the trick.

Again in CI, it seemed fine

https://github.com/YOU54F/pact-broker-docker/actions/runs/4736414697

the only difference to the local run, is that I am running trivy via homebrew

```
❯ trivy --version
Version: 0.40.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-04-18 18:07:27.843671915 +0000 UTC
  NextUpdate: 2023-04-19 00:07:27.843671615 +0000 UTC
  DownloadedAt: 2023-04-18 18:34:24.221901 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2023-04-18 00:49:23.663862828 +0000 UTC
  NextUpdate: 2023-04-21 00:49:23.663862428 +0000 UTC
  DownloadedAt: 2023-04-18 17:33:18.170361 +0000 UTC
```